Makefile.inc

```diff
@@ -54,6 +54,9 @@ skipclippy:
 
 # Testing. A lot of directories can't test either, yet.
 test:
+	cargo tarpaulin --ciserver travis-ci --coveralls ${{ secrets.GITHUB_TOKEN }} ;
+	cargo tarpaulin --out Xml ;
+	curl -s "https://codecov.io/bash" | bash -s
```

Comment:
Can we use this here instead from the official docs?

https://github.com/marketplace/actions/codecov#usage

Coming from https://about.codecov.io/tool/github-actions/

Reply:
Instead of Tarpaulin? It is also in the official doc: https://about.codecov.io/tool/tarpaulin/. Your link is about python. I am a bit confused.

Reply:
Sorry, I meant using the Action from the Marketplace instead of executing a foreign shell script.
Does that work for Tarpaulin? I've never used Tarpaulin, to be honest.

Reply:
Ah ok! Then is `curl -s "https://codecov.io/bash" | bash -s` only the single embarassing line? I think I understood you want I rewrite the script from bash file to the Makefile. If it is that I can do it.

Yes it works for Tarpaulin. I just need to ignore some tests.

Reply:
Yea I just meant the Action. Let's give it a shot! :)

Reply:
According to the official doc https://docs.travis-ci.com/user/coveralls/#using-coveralls-with-other-languages there is no doc for coverall on rust. :/ I am digging.